CRM/UF/Page/ProfileEditor.php

```diff
@@ -50,7 +50,7 @@ public static function registerProfileScripts() {
             'phoneType' => CRM_Core_PseudoConstant::get('CRM_Core_DAO_Phone', 'phone_type_id'),
           ],
           'initialProfileList' => $ufGroups,
-          'contactSubTypes' => CRM_Contact_BAO_ContactType::subTypes(),
+          'contactTypes' => CRM_Contact_BAO_ContactType::subTypes(),
```

Comment:
@eileenmcnaughton why the change of key here looks odd

Reply:
@seamuslee001 agree it looks wrong - changed it back